src/pygbag/filtering.py

```diff
@@ -27,13 +25,17 @@
 /ATTIC
 """.strip().split(
     "\n"
-)
+)  # Why not .splitlines()
 
```

Comment:
splitlines() seems to have better cross platform support and micropython supports it when MICROPY_PY_BUILTINS_STR_SPLITLINES is on , so go for it !